i18n/de.ts

```diff
@@ -0,0 +1,533 @@
+    </message>
+    <message>
+        <location filename="../main_dialog.py" line="317"/>
+        <source>Are you sure you want to remove the server configuration &apos;{selected_server_config}&apos;?</source>
+        <translation>Wollen Sie die Serverkonfiguration &apos;{selected_server_config}&apos; wirklich entfernen?</translation>
+    </message>
+    <message>
+        <location filename="../main_dialog.py" line="440"/>
+        <source>
+                    WMS already existed as a Mapbender source(s) and was successfully reloaded: {&apos;, &apos;.join(f&apos;#{i}&apos; for i in source_ids if i)}
+                    &lt;br&gt;&lt;br&gt;
+                    Link to Capabilities:
+                    &lt;br&gt;&lt;br&gt;
+                    &lt;a href = &quot;{wms_url}&quot; style = &quot;color: black; &quot; &gt; {wms_url} &lt;/a&gt;
+                    &lt;br&gt;&lt;br&gt;
+                    Link to Mapbender application:
+                    &lt;br&gt;&lt;br&gt;
+                    &lt;a href = &quot;{server_config.mb_basis_url}/application/{slug}&quot; style = &quot;color: black; &quot; &gt; {server_config.mb_basis_url}/application/{slug}&lt;/a&gt;
+                    </source>
+        <translation type="obsolete">
+                    WMS existierte bereits als Mapbenderquelle and wurde erfolgreich neu geladen: {&apos;, &apos;.join(f&apos;#{i}&apos; for i in source_ids if i)}
+                   &lt;br&gt;&lt;br&gt;
+                   Link zur Capabilities:
+                   &lt;br&gt;&lt;br&gt;
+                   &lt;a href = &quot;{wms_url)&quot; style = &quot;color:black; &quot; &gt; {wms_url} &lt;/a&gt;
+                   &lt;br&gt;&lt;br&gt;
+                    Link zur Mapbender Anwendung:
+                    &lt;br&gt;&lt;br&gt;
+                    &lt;a href = &quot;{server_config.mb_basis_url}/application/{slug}&quot; style = &quot;color: black; &quot; &gt; {server_config.mb_basis_url}/application/{slug}&lt;/a&gt;
+                    </translation>
+    </message>
+    <message>
+        <location filename="../main_dialog.py" line="464"/>
+        <source>
+                    WMS successfully created
+                    &lt;br&gt;&lt;br&gt;
+                    Link to Capabilities:
+                    &lt;br&gt;&lt;br&gt;
+                    &lt;a href = &quot;{wms_url}&quot; style = &quot;color: black; &quot; &gt; {wms_url} &lt;/a&gt;
+                    &lt;br&gt;&lt;br&gt;
+                    Link to Mapbender application:
+                    &lt;br&gt;&lt;br&gt;
+                    &lt;a href = &quot;{server_config.mb_basis_url}/application/{slug}&quot; style = &quot;color: black; &quot; &gt; {server_config.mb_basis_url}/application/{slug}&lt;/a&gt;
+                    </source>
+        <translation type="obsolete">
+                    WMS wurde erfolgreich erstellt
+                   &lt;br&gt;&lt;br&gt;
+                   Link zur Capabilities:
```

Comment:
Link to Capabilities: - en müsste sein: Link to GetCapabilities document:

DE dann Link zum GetCapabilities-Dokument:

Reply:
Das Dokument ist das Capabilities Dokument, ohne 'Get', analog zu getMap -> Map, GetFeatureInfo -> FeatureInfo, ...